iroha/src/lib.rs

```diff
@@ -295,8 +311,45 @@ impl Iroha {
         );
         Ok(())
     }
+
+    pub async fn waiting_for_peers_are_active(&self) -> Result<(), String> {
+        log::info!("waiting_for_peers_are_active");
+        let mut sorted_peers = Vec::new();
+        sorted_peers.push(Peer {
+            address: "iroha:8081".to_string(),
+        });
+        sorted_peers.push(Peer {
+            address: "iroha:8082".to_string(),
+        });
+        sorted_peers.push(Peer {
+            address: "iroha:8083".to_string(),
+        });
+        sorted_peers.push(Peer {
+            address: "iroha:8084".to_string(),
+        });
+        for peer in sorted_peers.iter() {
+            match Network::send_request_to(
+                &peer.address,
+                Request::new(uri::HEALTH_URI.to_string(), Vec::new()),
+            ).await? {
+                Response::Ok(_) => continue,
+                Response::InternalError => Err(format!(
+                    "Failed to send message - Internal Error on peer: {:?}",
+                    peer.address
+                ))?
+            }
+        }
+        log::info!("waiting_for_peers_are_active finished");
+        Ok(())
+    }
+
 }
 
+struct Peer {
```

Comment:
The `peer::Id` struct exists in `iroha_data_model` module, so there is no need to add it here.